contracts/api/IStaking.sol

```diff
@@ -0,0 +1,7 @@
+pragma solidity 0.8.6;
+
+interface IStaking {
+    function rolesOf(
+        address operator
+    ) external view returns (address, address, address);
```

Comment:
Can we name the parameters returned? Just to make it clear which one is beneficiary/authorizer/operator.

Reply:
Addressed 👍